blueprints/component-test/qunit-rfc-232-files/tests/__testType__/__path__/__test__.js

```diff
@@ -0,0 +1,36 @@
+<% if (testType === 'integration') { %>import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('<%= friendlyTestDescription %>', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
```

Comment:
> // Handle any actions with this.set('myAction', function(val) { ... });

should this be `this.on` ?

Reply:
no, because with the new system `this.on` does not exist anymore

@rwjblue correct me if I'm wrong

Reply:
found my own answer:
https://github.com/rwjblue/ember-qunit-codemod/blob/master/__testfixtures__/ember-qunit-codemod/on.input.js

https://github.com/rwjblue/ember-qunit-codemod/blob/master/__testfixtures__/ember-qunit-codemod/on.output.js

so in userland they have to define 
https://github.com/rwjblue/ember-qunit-codemod/blob/master/__testfixtures__/ember-qunit-codemod/on.output.js#L9..L12
and then 
```
this.actions.myAction = function() {}
```

Seems like a lot of code

Reply:
yeah, and it's only needed for old-style actions, with closure actions the thing I put in there is all that's needed

Reply:
got it.. :ok_hand: